packages/fast-components/src/slider-label/slider-label.ts

```diff
@@ -0,0 +1,58 @@
+import { attr, FastElement, observable } from "@microsoft/fast-element";
+import { FASTSlider } from "../slider";
+import { SliderOrientation } from "../slider/slider";
+
+export class SliderLabel extends FastElement {
+    @observable
+    public positionStyle: string;
+
+    @observable
+    public root: HTMLDivElement;
+
+    @attr
+    public max;
+
+    @attr
+    public position: string;
+
+    @attr
+    public label: string;
```

Comment:
Do any of these need to have `changed` events?

Reply:
max shouldn't have been there, position now has a changed event handler.